README.md

```diff
@@ -1,4 +1,14 @@
-# Centrifuge Pools Subquery
+# Centrifuge Subquery
+
+[![SubQuery](https://img.shields.io/static/v1?label=built+with&message=SubQuery&color=FF4581)](https://subquery.network/)
+[![embrio.tech](https://img.shields.io/static/v1?label=by&message=EMBRIO.tech&color=24ae5f)](https://embrio.tech)
+[![Centrifuge](https://img.shields.io/static/v1?label=for&message=Centrifuge&color=2762ff)](https://centrifuge.io/)
```

Comment:
Could you remove these lines? The subquery will be contributed to by EMBRIO, but of course not entirely built by EMBRIO either. This does imply the latter.

```suggestion
```